lua/astrocommunity/pack/rust/init.lua

```diff
@@ -9,7 +9,7 @@ return {
       config = {
         rust_analyzer = {
           settings = {
-            ["rust-analyser"] = {
+            ["rust_analyzer"] = {
```

Comment:
```suggestion
            ["rust-analyzer"] = {
```

I see now the original error. This should be a `-` though